app/models/challenge_claim.rb

```diff
@@ -48,11 +48,11 @@ def ==(other)
   scope :order_by_date, -> { order("created_at ASC") }
 
   def self.order_by_requesting_pseud(dir="ASC")
-    joins(REQUESTING_PSEUD_JOIN).order("pseuds.name #{dir}")
+    joins(REQUESTING_PSEUD_JOIN).order(["pseuds.name ?", dir])
   end
 
   def self.order_by_offering_pseud(dir="ASC")
-    joins(CLAIMING_PSEUD_JOIN).order("pseuds.name #{dir}")
+    joins(CLAIMING_PSEUD_JOIN).order(["pseuds.name ?", dir])
```

Comment:
Again, I believe this escapes the sort direction.

Reply:
`  def self.order_by_offering_pseud(dir="ASC")
    if dir == "ASC"
      joins(CLAIMING_PSEUD_JOIN).order("pseuds.name ASC")
    else
      joins(CLAIMING_PSEUD_JOIN).order("pseuds.name DESC")
    end
  end`

@tickinginstant that will work, happy with that approach ?

Reply:
Does `dir` need to be upcased for the comparison, or is it upcased before it gets passed in? Other than that, it looks like it should work.

(It might be nice to reduce the code duplication between the two branches of the if, but for code this short it's probably okay to leave it.)